pkg/controller/performanceprofile/components/kubeletconfig/kubeletconfig.go

```diff
@@ -15,11 +18,13 @@ import (
 )
 
 const (
-	cpuManagerPolicyStatic      = "static"
-	defaultKubeReservedCPU      = "1000m"
-	defaultKubeReservedMemory   = "500Mi"
-	defaultSystemReservedCPU    = "1000m"
-	defaultSystemReservedMemory = "500Mi"
+	cpuManagerPolicyStatic       = "static"
+	memoryManagerPolicyStatic    = "Static"
+	defaultKubeReservedCPU       = "1000m"
+	defaultKubeReservedMemory    = "500Mi"
+	defaultSystemReservedCPU     = "1000m"
+	defaultSystemReservedMemory  = "500Mi"
+	defaultHardEvictionThreshold = "100Mi"
```

Comment:
it would be nice to document (a comment is fine) how we determined this value.
Future PRs should retroactively add explanations for the other existing defaults.

Reply:
Some interesting insights regarding reserved memory:
- https://docs.openshift.com/container-platform/4.7/nodes/nodes/nodes-nodes-resources-configuring.html
```
kube-reserved | This setting is not used with OpenShift Container Platform. Add the CPU and memory resources that you planned to reserve to the system-reserved setting.
-- | --
system-reserved | This setting identifies the resources to reserve for the node components and system components. The default settings depend on the OpenShift Container Platform and Machine Config Operator versions. Confirm the default systemReserved parameter on the machine-config-operator repository.
```
- setting the default under the machine-config-operator, https://github.com/openshift/machine-config-operator/blob/fbc4d8e46a7746442f4de3651113d2181d458b12/templates/common/_base/files/kubelet-auto-sizing.yaml

@fromanirh Probably we need a separate PR to reflect these findings. WDT?